BlargBucket/ViewModels/ReadMeViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ReadMeViewModel.swift
+//  BlargBucket
+//
+//  Created by Nick Smillie on 12/23/14.
+//  Copyright (c) 2014 Luke Deniston. All rights reserved.
+//
+
+import Foundation
+
+class ReadMeViewModel: NSObject {
+	
+	dynamic var readMeMarkDown: String?
+	var readMeAttributed: NSAttributedString?
+	let repo = AppDelegate.sharedInstance().activeRepo
```

Comment:
Haven't removed the singleton yet - couldn't figure out how to get all the things working in the view controller when I started passing the repo in the init.